playwright/tests/page-objects/variables-editor.ts

```diff
@@ -12,43 +12,30 @@ export class VariablesEditor extends Editor {
   }
 
   override async isViewVisible() {
-    const header = this.viewFrameLoactor().locator('vscode-data-grid-cell:has-text("Key")');
+    const header = this.viewFrameLoactor().locator(
+      "//div[contains(@class, 'master-toolbar')]/div/div[contains(text(), 'Variables Editor')]"
+    );
     await expect(header).toBeVisible();
   }
 
   async hasKey(key: string) {
-    const field = this.viewFrameLoactor().locator(`vscode-text-field[grid-column="1"][current-value="${key}"]`);
+    const field = this.viewFrameLoactor().locator(`//td/div/span[contains(text(), '${key}')]`);
     await expect(field).toBeVisible();
   }
 
   async hasValue(value: string) {
-    const field = this.viewFrameLoactor().locator(`vscode-text-field[grid-column="2"][current-value="${value}"]`);
+    const field = this.viewFrameLoactor().locator(`//td/div[contains(text(), '${value}')]`);
```

Comment:
not such a big fan of xpath locators...
I think it should also be possible to write this with CSS selectors